CMakeLists.txt

```diff
@@ -249,6 +250,19 @@ if(PCAPPP_TARGET_COMPILER_CLANG
   add_compile_options(-Wall)
 endif()
 
+if (PCAPPP_BUILD_REPRODUCTIBLE)
+  add_definitions(-DPCAPPP_BUILD_REPRODUCTIBLE)
+  # We should not use __DATE__ nor __TIME__ in case of reproductible build
+  add_compile_options($<$<BOOL:${PCAPPP_TARGET_COMPILER_GCC}>:-Wdate-time>)
```

Comment:
This is only for GCC. Could you also add support for MSVC and Clang?

Reply:
Tested with Clang and GCC.
But not with MSVC